### DISCUSSION TITLE
TripXplorer - A trip app which will guide us

### DISCUSSION THREAD

**Discussion by anburocky3** [anburocky3/anburocky3#2] [Ideas]
**Problem?**
 Now-a-days everyone is feeling so stressed, drowning on depression due to various reasons. Mentality and being happiness is a different kind of therapy we all need to get rid of those problems. We can google it and find all places right? 
 > Not exactly, because google search engines will give you results not experience. And other third party websites who sounds like famous places are all clumpsy.  I like to propose a simple but effective app called "TripXplorer" (titled for now.)
 
 **Solution to the rescue:**
 We need a simpler app with the following workflow.
 1. Display all Indian states with relevant information.
 2. Choose the activity you like to begin. 
 (Friends, Family, Chillers, etc.)
 3. Ask, How much time you like to spend?
 4. Ask, How much can you afford per person?
5. Based on the above inputs, we can display relevant info..
6. Each place should have valid information like GPS, images, videos, ticket info, parking facility, food facility, etc.
7. Also based on the transport you choose, it has to calculate the trip cost.

**Comment by Jeya-rosini** [comment]
@anburocky3 -  I have some points to add on:
### Display nearby useful and emergency places

  For each searching place along with hotel, restaurants, link with nearby petrol bulk , repair station and hospitals

> Along With the distance for the particular place, the distance of the bulk, hospital should be shown below through map

### Share the experience

Like a feedback, we should get a good and bad experience about a place, hotel can be get through an input

> - We  should get inputs like place they visit through us, hotel they eat or stay, road condition traffic at that particular type.
> - Through that input we should get all the good experience like climate, weather and bad experience like traffic, poor quality food and stay in particular place
> - It can be updated in the website, others can view through that and decide their stay

**Comment by danielace1** [comment]
@anburocky3 
In my perspective :
# Mode of Travel
> First, we ask how you want to travel—like by bike, car, bus, train, or plane. Then, we show you how much it might cost to go to different places using your chosen mode of transportation. 
- Once they've made their choice, we have to show not only the estimated costs for traveling to different destinations but also tailors its suggestions based on their selected mode of transportation. For instance, if they opt for driving, the app will highlight nearby scenic routes, pit stops, and attractions along the way. 

> The travel planning service understands how frustrating it is when people aren't sure how much a trip will cost. That's why it's focused on giving clear price information. This makes planning easier and less stressful. Their aim is to make travel planning simple and worry-free. By showing costs upfront, they want travelers to feel more confident about their trips.

**Comment by GayathriHG** [comment]
In my perspective ,
- we can divide the places available into two 
1. tourist places :
like: monuments, adventure parks.
3. unexplored places :
like: trekking spots, parks, or nearby remote villages , where they can explore the culture of the area.

- and also , notifying the people of rental bikes nearby so that they can explore places with a private vehicle.
